tests/data/point-cloud/README.md

```diff
@@ -0,0 +1,10 @@
+Data Sources
+============
+
+#### `auckland`
+
+[Auckland, New Zealand 2013](https://portal.opentopography.org/datasetMetadata?otCollectionID=OT.062016.2193.1); Land Information New Zealand; CC-BY
+
+#### `autzen.las`
+
+[Autzen Stadium](https://github.com/PDAL/data/tree/master/autzen); Watershed Sciences, Inc; BSD
```

Comment:
https://github.com/PDAL/data/blob/master/LICENSE says CC-BY 4.0 International, not BSD. Is there a more specific license file somewhere?

Reply:
https://pdal.io/copyright.html says BSD but I guess it would be less specific than this one. I'll change it to CC-BY